addon/components/bourbon-modal.js

```diff
@@ -20,6 +20,10 @@ export default Component.extend({
     )} btw-mr-3`;
   }),
 
+  showCloseButton: computed('modalService.dissmissable', 'modalService.notificationModal', function() {
+    return this.get('modalService.dissmissable') || !this.get('modalService.notificationModal')
+  }),
```

Comment:
Not sure where else this might appear, but could you fix the spelling from `dissmissable` to `dismissable`?

Reply:
oops :)